### DISCUSSION TITLE
Slurm-web v4 on CentOS7

### DISCUSSION THREAD

**Discussion by dai542** [rackslab/Slurm-web#422] [Q&A] [answered]
How can I use Slurm-web v4 on CentOS 7? I couldn’t find a [repository](https://pkgs.rackslab.io/rpm) suitable for CentOS 7. Due to various reasons, we are not planning to upgrade the CentOS 7 system at the moment. Are there any alternative solutions?

**Comment by rezib** [comment, marked as answer]
Hello @dai542, I just converted the issue you opened #421 into this discussion as this is more a question than an actual bug. We don't support CentOS 7 yet. I see 3 possibles options:

1. You can order to Rackslab the development of officially supported RPM packages for CentOS 7,
2. You can order to Rackslab the development and distribution of docker/podman containers (see #266),
3. You can try one of the above options on your own.

If you want a quote for 1. or 2., feel free to [ask in our contact form](https://rackslab.io/en/contact/).
